### PR TITLE
Update to latest nixpkgs and poetry2nix

### PR DESCRIPTION
**Pull request by zupo**
The title says it all.

### CODE REVIEW

**Comment by zupo** [comment]
I have read the CLA Document and I hereby sign the CLA

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -11,6 +11,8 @@ jobs:
     - uses: cachix/install-nix-action@v16
       with:
         nix_path: nixpkgs=channel:nixos-unstable
+    - name: tet
```
Comment (domenkozar) [inline .github/workflows/ci.yml:14]: What's this?

Comment (zupo) [reply .github/workflows/ci.yml:14]: testing, work-in-progress

**Comment by zupo** [comment]
@domenkozar: ready for review

**Pull request merged**